packages/turbo-ignore/package.json

```diff
@@ -24,15 +24,18 @@
     "check-types": "tsc --noEmit"
   },
   "devDependencies": {
-    "typescript": "^4.7.4",
-    "turbo-utils": "workspace:*",
-    "tsup": "^5.12.1",
-    "tsconfig": "workspace:*",
-    "ts-jest": "^27.1.1",
-    "jest": "^27.4.3",
-    "eslint": "^8.20.0",
-    "@types/node": "^16.11.12",
+    "@manypkg/find-root": "^1.1.0",
     "@types/jest": "^27.4.0",
-    "@manypkg/find-root": "^1.1.0"
+    "@types/node": "^16.11.12",
+    "eslint": "^8.20.0",
+    "jest": "^27.4.3",
+    "ts-jest": "^27.1.1",
+    "tsconfig": "workspace:*",
+    "tsup": "^5.12.1",
+    "turbo-utils": "workspace:*",
+    "typescript": "^4.7.4"
+  },
+  "dependencies": {
+    "meow": "^7.1.1"
```

Comment:
I would prefer we write this one manually to avoid the extra dependency. Since this is always run via `npx` and in `CI`, install time matters here so we can bail ASAP when needed. 

Also since this will only be run in CI, I don't see the need to add a full CLI parsing library when we will only ever need one or two commands.